src/mame/drivers/pg685.cpp

```diff
@@ -105,7 +105,10 @@ class pg685_state : public driver_device
 		m_maincpu(*this, "maincpu"),
 		m_vram(*this, "framebuffer"),
 		m_vram16(*this, "framebuffer16"),
-		m_fontram(*this, "charcopy")
+		m_fontram(*this, "charcopy"),
+		m_fdc (*this, "fdc"),
```

Comment:
Can you please get rid of the extra GNU-style space?  It's inconsistent with the lines immediately above and below it.